jib-gradle-plugin/README.md

```diff
@@ -74,32 +83,59 @@ jib.to.image = 'my-docker-id/my-app'
 Build your container image with:
 
 ```shell
-gradle build jib
+gradle jib
 ```
 
 Subsequent builds are much faster than the initial build. 
 
-If you want to clear Jib's build cache and force it to re-pull the base image and re-build the application layers, run:
+*Having trouble? Let us know by [submitting an issue](/../../issues/new), contacting us on [Gitter](https://gitter.im/google/jib), or posting to the [Jib users forum](https://groups.google.com/forum/#!forum/jib-users).*
+
+#### Build to Docker daemon
+
+Jib can also build your image directly to a Docker daemon. This uses the `docker` command line tool and requires that you have `docker` available on your `PATH`.
 
 ```shell
-gradle clean build jib
+gradle jibDockerBuild
 ```
 
-*Having trouble? Let us know by [submitting an issue](/../../issues/new), contacting us on [Gitter](https://gitter.im/google/jib), or posting to the [Jib users forum](https://groups.google.com/forum/#!forum/jib-users).*
+##### Minikube Docker daemon
```

Comment:
I think I agree with Chanseok's old comment about the sizing of this subtitle being weird. The way the titles shrink as you go down seems strange, even if they are sub-sections of each other. Maybe you can just remove this title and combine the minikube section with the docker daemon section.